src/_tkagg.cpp

```diff
@@ -67,6 +67,11 @@ static PyObject *mpl_tk_blit(PyObject *self, PyObject *args)
         PyErr_SetString(PyExc_ValueError, "Failed to extract Tk_PhotoHandle");
         goto exit;
     }
+    if (0 > y1 || y1 > y2 || y2 > height || 0 > x1 || x1 > x2 || x2 > width) {
```

Comment:
```suggestion
    if (y1 < 0 || y1 > y2 || y2 > height || x1 < 0 || x1 > x2 || x2 > width) {
```
I find it easier this way "y1 is smaller than 0 or larger than y2". `0 > y1` sort of ties a knot in my brain. :smile:

Reply:
That is fair, but having all of the `>` go the same way is also helpfule...

Moot now that @efiring merged it..